test/integration-test/src/tests/stack_argument.rs

```diff
@@ -0,0 +1,84 @@
+use aya::{
+    include_bytes_aligned, maps::AsyncPerfEventArray, programs::UProbe, util::online_cpus, Bpf,
+};
+use aya_log::BpfLogger;
+use bytes::BytesMut;
+use log::warn;
+use tokio::task;
+
+use crate::STACK_ARGUMENT;
+
+pub struct Args {
+    a_0: u64,
+    a_1: u64,
+    a_2: u64,
+    a_3: u64,
+    a_4: u64,
+    a_5: u64,
+
+    a_6: u64,
+    a_7: i64,
+}
+
+#[no_mangle]
+#[inline(never)]
+pub extern "C" fn trigger_stack_argument(
+    a_0: u64,
+    a_1: u64,
+    a_2: u64,
+    a_3: u64,
+    a_4: u64,
+    a_5: u64,
+    //from arg6, stack_argument would be used
+    a_6: u64,
+    a_7: i64,
+) {
+}
+
+#[tokio::test]
+async fn stack_argument() {
+    event_logger::init();
+    let mut bpf = Bpf::load(crate::STACK_ARGUMENT).unwrap();
+
+    if let Err(e) = BpfLogger::init(&mut bpf) {
```

Comment:
do we need the logs here? this looks like leftover debugging code; our tests don't usually consume logs (except for the logging test).